phpunit.xml.dist

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./vendor/autoload.php" colors="true" strict="true">
+<phpunit bootstrap="./vendor/autoload.php" colors="true">
```

Comment:
An improvement would be to apply these changes: https://github.com/SymfonyTest/SymfonyDependencyInjectionTest/pull/110/files#diff-98c12a693941ae3c4f50872403416471

Reply:
Nice. Will fix first thing Monday!

Reply:
I can make a PR on your PR so you just have to click merge !

Reply:
Here you go, please merge : https://github.com/emodric/SymfonyDependencyInjectionTest/pull/1

Reply:
Done!